packages/p2p/src/components/my-ads/form-ads.jsx

```diff
@@ -65,181 +65,183 @@ class FormAds extends Component {
-                                                    type='number'
-                                                    error={touched.fix_price && errors.fix_price}
-                                                    label={localize('Fixed price')}
-                                                    className='my-ads__form-field'
-                                                    trailing_icon={<span className='my-ads__form-field--trailing'>{`${values.currency}/${values.asset}`}</span>}
-                                                    required
-                                                />
-                                            )}
-                                        </Field>
-                                        <Field name='amount'>
-                                            {({ field }) => (
-                                                <Input
-                                                    {...field}
-                                                    data-lpignore='true'
-                                                    type='number'
-                                                    error={touched.amount && errors.amount}
-                                                    label={localize('Amount')}
-                                                    className='my-ads__form-field'
-                                                    trailing_icon={<span className='my-ads__form-field--trailing'>{values.asset}</span>}
-                                                    required
-                                                />
-                                            )}
-                                        </Field>
-                                    </div>
-                                    <div className="my-ads__form--container">
-                                        <Field name='min_transaction'>
-                                            {({ field }) => (
-                                                <Input
-                                                    {...field}
-                                                    data-lpignore='true'
-                                                    type='number'
-                                                    error={touched.min_transaction && errors.min_transaction}
-                                                    label={localize('Min. transaction')}
-                                                    className='my-ads__form-field my-ads__form-field--single'
-                                                    trailing_icon={<span className='my-ads__form-field--trailing'>{values.currency}</span>}
-                                                    required
-                                                />
-                                            )}
-                                        </Field>
-                                        <Field name='max_transaction'>
+        return (
+            <Fragment>
```

Comment:
could we not make this change in this PR? it makes the diff hard to review, we can create another PR for cleaning up later

Reply:
eslint failed if i dont do this